MainActivity.java

```diff
@@ -0,0 +1,112 @@
+package com.example.service;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+
+    public static final String BROADCAST_ACTION = "com.06032019.demo";
+
+    Button startServiceBt;
+    TextView receiverTv;
+    MyBroadCastReceiver myBroadCastReceiver;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        initializeMembers();
+        setListener();
+        registerMyReceiver();
+    }
+
+    private void initializeMembers()
+    {

```

Comment:
Please use Java code style rules.

Reply:
changed: https://github.com/NewAzgardec/Calculator/commit/8bed678c60920d4f7e99ed232d4c9dfbebe5d79b
@ajaxspace

Reply:
Please check https://google.github.io/styleguide/javaguide.html. Rules about bracers. Android studio will do good formatting just use reformat code option.

Reply:
thanks, I understood, I will try not to repeat this mistake again:
https://github.com/NewAzgardec/Calculator/commit/68140a841a549ea6344b4eeedd8941cefd4ccee9